content/markdown/02-working-with-data/02-01-reading-writing.rmd

```diff
@@ -0,0 +1,147 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Reading and Writing Files in R 
+**Author:** Cole Brookson
+**Date:** 13 June 2022
+
+Quickly after beginning to use R, as a biologist it is likely you will want to analyze some data. These data will be often stored as external files in your computer, with the most common data file types being Excel Spreadsheet files (`.xls(x)`), Comma Separated Value files (`.csv`), or Plain Text (`.txt`) files.
+
+Since it is best practice to **never** directly edit your raw data files (because what if you need to go back to the original version!?), a common workflow is to:
+
+1. *Read your data into R *`
+
+2. Perform some kind of analysis, creating new data objects, figures, or outputs, then 
+
+3. *Write out those data objects/figures/outputs to your computer for later use*
```

Comment:
I might say "save" rather than "write out"? Not committed to that, but write out felt confusing to me.